v4/source/teams.yaml

```diff
@@ -784,6 +784,57 @@
         '404':
           $ref: '#/responses/NotFound'
 
+  '/teams/{team_id}/members/{user_id}/schemeRoles':
+    put:
+      tags:
+        - teams
+      summary: Update the `scheme_admin` / `scheme_user` properties of a team member.
```

Comment:
Is there a shorter way to say this? The summaries show up in the sidebar so shorter is better where possible